main.rb

```diff
@@ -1,49 +1,50 @@
 require './app'
+require './menu'
+require_relative 'Creators/person_creator'
+require_relative 'Creators/book_creator'
+require_relative 'Creators/rental_creator'
+
+class Main
+  def initialize
+    @app = App.new
+  end
+  include Menu
 
-def main
-  app = App.new
-
-  puts 'Welcome to School Library App!'
-  puts ''
-
-  loop do
-    display_menu
-    option = gets.chomp
+  def choose_option
+    loop do
+      display_menu
+      option = gets.chomp.to_i
+      break if option == 7
 
-    if option == '7'
-      puts 'Thank you for using this app!'
-      break
+      execute_options(option)
     end
-
-    options(option, app)
   end
-end
 
-def options(option, app)
-  case option
-  when '1' then app.list_books
-  when '2' then app.list_people
-  when '3' then app.new_person
-  when '4' then app.new_book
-  when '5' then app.new_rental
-  when '6' then app.list_rentals
-  else
-    puts 'That is not a valid option'
+  def execute_options(option)
+    case option
+    when 1
+      @app.list_books
+    when 2
+      @app.list_people
+    when 3
+      @app.new_person
```

Comment:
- [ ] As required for the project, kindly ensure that your library console app still behaves in the same as the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ). Right now when I want to create a student, I get the below message:
![Screenshot from 2023-04-17 17-22-49](https://user-images.githubusercontent.com/83042742/232534838-ca832e78-bd1e-4f84-acb8-b300b5aa5c0f.png)